src/App.css

```diff
@@ -771,6 +791,9 @@ button {
   display: flex;
   align-items: center;
   gap: 14px;
+  padding: 15px 30px;
+  background-color: #ffffff;
+  max-height: 70px;
```

Comment:
Remove these lines they ruin the skill-name styles and are unrelated to this PR.